### ISSUE TITLE
Make options more user friendly

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-cli#159]
Multi-word options, like "show module", uses underscore "_" instead of hyphen "-" as a substitution for space. I.e.:

```
--show_module
--no-show_module
```

It would be more user-friendly with 

```
--show-module
--no-show-module
```

Since underscore has been used for a long time, I suggest that both are supported.

**Comment by matsduf** [comment]
@vlevigneron, would this be simple or is it a big change?

**Comment by matsduf** [comment]
Resolved by #212.
